src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -165,8 +191,8 @@ class boss_bronjahm : public CreatureScript
                             me->CastSpell(me, SPELL_SOULSTORM, false);
                             break;
                         case EVENT_FEAR:
-                            if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 1, 0.0f, true))
-                                me->CastCustomSpell(SPELL_FEAR, SPELLVALUE_MAX_TARGETS, 1, target, false);
+                            if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 0, 0.0f, true)) // Doesn't he fear just one enemy?
```

Comment:
OK, I will start by reverting to the original line: `SELECT_TARGET_RANDOM, 1, 0.0f, true`